components/Elevation/src/MDCElevation.h

```diff
@@ -0,0 +1,31 @@
+// Copyright 2019-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import <CoreGraphics/CoreGraphics.h>
+#import <Foundation/Foundation.h>
+
+/**
+ This protocol is used by @c UIViews that want to provide their elevation and wish to react to
```

Comment:
Nit: Please replace this sentence with something more like, "Provides APIs for views to communicate their relative and absolute elevation."

Reply:
Updated.